Shuttle (New): CLU Geode - Scrapyard Trader's Vessel

LoakaOfTheWind opened the pull request:
## About the PR
Adds the CLU Geode, a trading vessel built into a rock. Its rocky exterior will help conceal all of your Certifiably Legitimate Undertakings!

## Why / Balance
The Geode is designed, first and foremost, for the trader with the desire to be a bit more subtle. The design of the vessel has it hidden away inside an asteroid, enough to make it seem at least at first glance like an asteroid itself. Its main purpose is to enhance trading RP, of both the legitimate and non-legitimate kind, without necessarily pigeon-holing it into one role.

The Geode features:
- A pair of airlocks either side (which are also its main 'tell' on radar) to conduct business with multiple people at once.
- A rear loading ramp for larger cargo or swiftly disposing off unwanted or incriminating objects.
- A bar area for greasing the wheels of commerce and/or law.
- A convenient hidey-hole for stashing your valuables.
- A decently sized office area on the bridge, with room for you, your pets, your paperwork, and maybe an extra console (hint hint, there's no IFF console on this thing by default)
- A salvage techfab, some salvage EVA suits, and a material reclaimer to assist in maintaining your vessel during clandestine meetings.
- Shutters on all external windows, for when you need to go dark (disclaimer: boarded up shutters don't close! new owners will need to put that on their maintenance list. Gods bless the Scrapyard! These are controlled from the yellow bordered buttons)
- A button to bolt the exterior doors. A feature from a paranoid former pilot. (controlled from the red bordered button on the bridge)

Specs:
- 2 S.U.P.E.R.P.A.C.M.A.N. Generators
- 13 thrusters (3 fore, 2 each side, 6 aft)
- 1 Gyroscope

## How to test
- Head to your local scrapyard and purchase the CLU Geode
- Try to close the shutters
- Stick all of your ill-gotten gains in the hidey-hole
- Try not to trip over the empty bottles

## Media
Normal zoom, full lights
![geodelights](https://github.com/user-attachments/assets/1debb01d-9cfe-4c48-87d5-b4ed35e8fb7f)

Without lights
![geodenolights](https://github.com/user-attachments/assets/e431396f-7708-40c2-9d51-33b987b2accc)

Subfloor enabled
![geodesubfloor](https://github.com/user-attachments/assets/afb8d2bd-2744-45e4-8672-46e629244a68)



## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [ ] I have read [CONTRIBUTING.md](https://github.com/new-frontiers-14/frontier-station-14/blob/master/CONTRIBUTING.md) and and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ ] I have added media to this PR or it does not require an ingame showcase.
- [ ] I have reviewed the [Ship Submission Guidelines](https://frontierstation.wiki.gg/wiki/Ship_Submission_Guidelines) if relevant.
- [ ] I confirm that AI tools were not used in generating the material in this PR.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->

Houtblokje commented:
I do wonder, what is the RP reasoning behind this ship? Is it a ship crashed into an asteroid? Is it an asteroid with a ship built inside of it by a madman? Something else? How do these elements play into your design?
I am just curious to know what the lore behind this ship is.

LoakaOfTheWind commented:
how did i bug it HARDER?!

LoakaOfTheWind commented:
Alrighty, I'm going to make the changes post and then upload the commit. Let's go piece by piece.

The Bar:
![image](https://github.com/user-attachments/assets/c4af4df9-f029-4f2e-aebd-03ea8b4f7448)
Added the extra window, added a shutter and linked that. Also added a second serving counter where the tinted window was. Now the bartender can join in all the goblin games while still mixing drinks! Could even work as a dealer from there. The moment someone adds a goblin plush, I'll replace that dastardly moth bartender! Looking so cute and dapper in its little outfit ...

Common Room:
![image](https://github.com/user-attachments/assets/c1596207-82c6-4bb9-bc8a-e331903f3029)
Shoved the filing cabinet over. The police will never find my secret supply of limes now. Also, an air alarm in here, linked appropriately. While we're here, the maints doors have been replaced with proper external airlock doors.

Bridge:
![image](https://github.com/user-attachments/assets/4db31a23-e31b-42bd-8d08-7ef190809516)
Station records moved over. Only have to lean over the dog to reach it. Air alarm added in there too - I checked you can reach all of these just fine. Yellow button now closes shutters inside the bridge only, while the red button bolts all the external airlocks. All the other shutter+bolt buttons now just do the shutters. I reckon if you really want to bolt internally, we have the shipyard remote for that.

Big Picture:
![image](https://github.com/user-attachments/assets/a0b6fad8-baf0-409f-9451-c2f2bd3c589b)
Wiring redone, I think MV is only ever in a wall to touch an APC, and the worst offenders are the necessary-to-reach-thrusters LV cables, of which I've toned down a few there by having them go under thrusters instead of through rock/wall.
Lower loading platform is now a dock, built to the same kind of specs as the rest. It does make the radar outline look a lot more flat at the back but trying to 'texture' it back there would just make docking a pain. As a consequence of the airlock being there, the engineering room got a single extra tile of space (the reclaimer has been moved one tile south, making the shutter button even easier to reach, as well as the canisters). Also makes it a bit more open for installing RTGs or whatever.

Also, I definitely did NOT forget to put emergency lights in. Again. I did NOT install them just as I was about to finish this post. Honest.
![image](https://github.com/user-attachments/assets/387373a6-864d-4c65-9cc4-3511d70bdd8a)

LoakaOfTheWind commented:
Redid wiring per your suggestions, works perfectly. Dock has come out one tile, also lets it be identical to the other two airlocsk (with the second window) which is nice for my brain. Price has dropped to 52,000 - Appraise at 47,214, +10% is just shy of 52k so it got rounded up slightly.
Will have a crack at a guidebook as soon as I figure out how!
